docs/installation.rst

```diff
@@ -20,6 +20,38 @@ There are different ways to install |project_name|:
     have the latest code or use revision control (each release is
     tagged).
 
+.. _installation-requirements:
+
+Pre-Installation Considerations
+-------------------------------
+
+|project_name| Repository Filesystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- No special filename requirements exist. Filenames may contain any valid
+  ascii character and be of any length. |project_name| stores data only 3
+  directory levels deep and uses short directory and filenames.
+- No special file metadata requirements exist regarding permissions or extended
+  file attributes. |project_name| only requires read/write permissions within
+  the repository filesystem.
+- Segment files generated by |project_name| are no larger than 5MB by default
```

Comment:
as your PR is against master branch, it needs to be 500MiB.
5MiB is 1.0-maint branch.